### PR TITLE
[fix bug 1136934] Firefox 38.0.5 whatsnew page

### PR DESCRIPTION
**Pull request by alexgibson**
Three different templates depending on the locale:

* Video plus Pocket - `/en-US/firefox/38.0.5/whatsnew/`
* Video only - `/fr/firefox/38.0.5/whatsnew/`
* Pocket only - `/ja/firefox/38.0.5/whatsnew/`

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-base.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% add_lang_files "firefox/38_1_whatsnew" %}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-base.html]: If we need it, I'd use the same ```firefox/whatsnew_38```

Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-base.html]: Thanks, meant to update this one when I renamed things

File: bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-a.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+{% block body_id %}whatsnew-38-a{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block content %}
+<main role="main">
+  <section id="video-content">
+    <h1 class="large">{{_('From our hands to yours')}}</h1>
+    <p class="tagline">{{_('Firefox is built from the heart by thousands who believe in personal freedom on the Web.')}}</p>
+    <a id="video-thumbnail" href="{{ fx_38_video_url}}.mp4">{{_('Watch Video')}}</a>
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-a.html]: Can we use ```Watch the video```, which is the string we used so far and already available in main.lang?

Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-a.html]: Sure +1

**Comment by alexgibson** [comment]
Still waiting on the final localized videos, plus poster image - but this should now be good for code review.

**Comment by alexgibson** [comment]
Still missing de, es-MX, and pt-BR videos.

**Comment by ckprice** [comment]
All videos and poster image are [posted](https://bugzilla.mozilla.org/show_bug.cgi?id=1140042).

**Comment by ckprice** [comment]
@jpetto r?

File: bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-pocket-video.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+{% block site_css %}
+  {% stylesheet 'firefox_whatsnew_38_pocket' %}
+{% endblock %}
+
+{% block page_title %}{{ _('Pocket comes to Firefox!') }}{% endblock %}
+{% block body_id %}whatsnew-38-pocket-video{% endblock %}
+
+{% block content %}
+<main role="main">
+    <section id="pocket">
```
Comment (jpetto) [inline bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-pocket-video.html]: Nit - extra indentation.

File: media/js/firefox/whatsnew_38/whatsnew-video.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(function(){
```
Comment (jpetto) [inline media/js/firefox/whatsnew_38/whatsnew-video.js]: Do these need to be in a `document.ready` handler?

Comment (alexgibson) [inline media/js/firefox/whatsnew_38/whatsnew-video.js]: Maybe not, but I think there's very little gain here for this page to make them immediately invoked?

**Comment by jpetto** [comment]
Since we're using the same video poster image here and in PR #2932, we should probably merge the images to master first, then rebase the PRs. I can open a PR for the images.

File: media/js/firefox/whatsnew_38/whatsnew-video.js (hunk 4 above)
Comment (jpetto) [inline media/js/firefox/whatsnew_38/whatsnew-video.js]: No big gain, but one less event handler and keeping consistent with other JS files.

Comment (alexgibson) [inline media/js/firefox/whatsnew_38/whatsnew-video.js]: Hmmm.... okay you persuaded me :+1:

**Comment by alexgibson** [comment]
@jpetto - rebased and using common poster images for video

**Comment by jpetto** [comment]
Looking good. Squashing please?

**Comment by alexgibson** [comment]
Squarshed!

**Comment by alexgibson** [comment]
@jpetto - rebased and resolved conflicts. I renamed the function in `/firefox/views.py` to `show_38_0_5_firstrun_or_whatsnew` and updated accordingly.

**Comment by jpetto** [comment]
r+! :haircut:

**Pull request merged**